Assets/Prefabs/Core/Manager.prefab

```diff
@@ -2710,6 +2710,7 @@ MonoBehaviour:
   - SecondCutscene
   - Level2InterstitialCutscene
   - EndCutscene
+  - MainMenu
```

Comment:
wha?

Reply:
this doesnt display in unity since the cutscene list was deleted, not a "unapplied override" either
shrug

Reply:
could also just revert the change lol

but whatevs